_pages/Jobs

```diff
@@ -0,0 +1,37 @@
+---
+title: "Jobs"
+permalink: /jobs/
+excerpt: "Creating and configuring jobs within a Skyve application."
+toc: true
+sidebar:
+  title: "Index"
+  nav: docs
+---
+ 
+Jobs within a Skyve application is where system jobs are created and edited. New jobs can be created using the modules within the application. System jobs can also be maintained here if they need to be updated over time. 
```

Comment:
This is a bit unclear. Maybe change to:

Jobs within a Skyve application allow long running tasks (jobs) to be scheduled to run in the background. Jobs can be scheduled to be run once, or a schedule can be saved if they need to run repeatedly over time, e.g. a nightly backup job.